services/bitbucket/bitbucket-pull-request.service.js

```diff
@@ -14,14 +15,63 @@ function pullRequestClassGenerator(raw) {
   const badgeSuffix = raw ? '' : ' open'
 
   return class BitbucketPullRequests extends BaseJsonService {
```

Comment:
One final nitpick 😄 Can we make the class name here consistent with the file name? I didn't realize the discrepancy (services class name ends with `requests` whereas file name ends with `request`) til trying to run the tests